plugins/pulp_docker/plugins/distributors/rsync_distributor.py

```diff
@@ -0,0 +1,112 @@
+from gettext import gettext as _
+import logging
+
+from pulp.common.config import read_json_config
+from pulp.plugins.distributor import Distributor
+from pulp.plugins.rsync import configuration as rsync_config
+
+from pulp_docker.plugins.distributors.publish_steps import DockerRsyncPublisher
+
+TYPES = []
+try:
+    from pulp_docker.common.constants import (IMAGE_TYPE_ID)
+    TYPES.extend([IMAGE_TYPE_ID])
```

Comment:
Actually, i am removing this try statement all together. This code was originally written to work with 2.7, but is not needed for 2.10.